tools/offline_log_viewer/storage.py

```diff
@@ -302,7 +302,9 @@ def __search(self):
             for topic in listdirs(join(self.base_dir, nspace)):
                 for part_ntp_id in listdirs(join(self.base_dir, nspace,
                                                  topic)):
-                    assert re.match("^\\d+_\\d+$", part_ntp_id)
+                    assert re.match("^\\d+_\\d+$", part_ntp_id), \
```

Comment:
nit: can you add a description in the commit message and precede the title with `tools:`

Reply:
Done :)